src/transformers/models/fsmt/modeling_fsmt.py

```diff
@@ -1122,23 +1122,6 @@ def prepare_inputs_for_generation(
             "use_cache": use_cache,  # change this to avoid caching (presumably for debugging)
         }
 
-    def adjust_logits_during_generation(self, logits, cur_len, max_length, **kwargs):
-        if cur_len == max_length - 1 and self.config.eos_token_id is not None:
-            self._force_token_ids_generation(logits, self.config.eos_token_id)
-        return logits
-
-    def _force_token_ids_generation(self, scores, token_ids) -> None:
```

Comment:
is this function equivalent to the other `_force_token_ids_generation` functions?

Reply:
cc @stas00

Reply:
Yes, and it's only called inside `adjust_logits_during_generation`, so it's safe to remove